src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return (days * 40) - 50;
+  } else if (days >= 3) {
```

Comment:
If you write `return` inside `if` block, the next `else` is useless.

With `return` structure should look like this:

```
if (condition1) {
  return x;
}

if (condition2) {
  return y
}

return z;
```